resources/views/api/openinghours/multiple_days.blade.php

```diff
@@ -0,0 +1,44 @@
+<div vocab="http://schema.org/" typeof="Library" class="openinghours openinghours--table">
+    <h1>Openinghours</h1>
+    @if($data[0]['openinghours'])
+        @foreach($data[0]['openinghours'] as $dayInfoObj)
```

Comment:
Use `@include` for common template output. (see https://laravel.com/docs/5.5/blade#including-sub-views).